test/modules/aws-sns-integration.ts

```diff
@@ -27,7 +31,33 @@ const installAll = runInstallAll.bind(null, dbAlias);
 const uninstall = runUninstall.bind(null, dbAlias);
 const region = defaultRegion();
 
-const modules = ['aws_sns'];
+const lambdaFunctionName = `${prefix}${dbAlias}`;
+const lambdaFunctionCode =
+  'UEsDBBQAAAAIADqB9VRxjjIufQAAAJAAAAAIABwAaW5kZXguanNVVAkAAzBe2WIwXtlidXgLAAEE9QEAAAQUAAAANcyxDoIwEIDhnae4MNFIOjiaOLI41AHj5NLUA5scV3K9Gojx3ZWB8R++H5c5iWb78vwkFDgD+LxygKFw0Ji4wTeythASKy5q4FPBFjkRWkpjU3f3zt1O8OAaDnDpr85mlchjHNYdcyFq4WjM3wpqEd5/26JXQT85P2H1/QFQSwECHgMUAAAACAA6gfVUcY4yLn0AAACQAAAACAAYAAAAAAABAAAApIEAAAAAaW5kZXguanNVVAUAAzBe2WJ1eAsAAQT1AQAABBQAAABQSwUGAAAAAAEAAQBOAAAAvwAAAAAA';
+// Base64 for zip file with the following code:
+// exports.handler =  async function(event, context) {
+//   console.log("EVENT: \n" + JSON.stringify(event, null, 3))
+//   return context.logStreamName
+// }
```

Comment:
nit- having the function as a string and encoding using the node.js std

Reply:
it's not just base64, it will need to zip it as well , needing an extra dependency. So i prefered to just leave it simple as it was only used here